### DISCUSSION TITLE
ETL failing after adding incremental timeseries

### DISCUSSION THREAD

**Discussion by SbiCA** [ravendb/ravendb#15193] [Q&A]
Hi, 

We've recently added a incremental timeseries to an existing document that was loaded in a ETL now the ETL is failing with an unsupported exception.

``` csharp
System.NotSupportedException: Load isn't support for incremental time series 'INC:EventMembersStats' at document '6675f947-6e6e-4994-a1fd-bc6c3b55eb1e'
   at Raven.Server.Documents.ETL.Providers.Raven.RavenEtl.LoadInternal(IEnumerable`1 items, DocumentsOperationContext context, EtlStatsScope scope) in C:\Builds\RavenDB-Stable-5.4\54012\src\Raven.Server\Documents\ETL\Providers\Raven\RavenEtl.cs:line 163
   at Raven.Server.Documents.ETL.EtlProcess`6.Load(IEnumerable`1 items, DocumentsOperationContext context, TStatsScope stats) in C:\Builds\RavenDB-Stable-5.4\54012\src\Raven.Server\Documents\ETL\EtlProcess.cs:line 461
```

Is there a way to ignore the timeseries in the ETL script and should this not be the default behaviour?

Cheers
Simon

**Comment by ayende** [comment]
You are using an ETL with no script, right?
You can workaround this with the following script: `loadToOrders(this)` 

Or control this directly using: https://ravendb.net/docs/article-page/5.4/csharp/server/ongoing-tasks/etl/raven#time-series-load-behavior-function

**Reply by SbiCA** [reply]
![image](https://user-images.githubusercontent.com/11393585/198878771-78941fea-0f93-4757-b8cb-5179dd300d89.png)

@ayende  Yes exactly pasted a sprint screen here.  Could this issue be cause due to having multiple collection in the same ETL but only one of them is using time series?

In our case:
- eventdocuments => has timeseries, counters
- all the others posts,memberships ... don't

**Reply by ayende** [reply]
For the `EventDocuments`, create its own ETL script. And use `loadToEventDocuments(this)`, which will send just the document, not the time series.

**Reply by SbiCA** [reply]
I did it like this since I need the counters as well

``` javascript
loadToEventDocuments(this);


function loadCountersOfEventDocuments(docId, counterName) {
   return true;
}
```

Seems to work like that 🎉  maybe that would be worth adding to the incremental timeseries docs for the time being as it's not obvious. 
Is there a plan to support incremental timeseries in ETL? since "normal" timeseries seem to be supported

Cheers

**Reply by garayx** [reply]
yes, we have a ticket opened to support incremental timeseries, https://issues.hibernatingrhinos.com/issue/RavenDB-17540/ETL-of-incremental-TS
